src/language-server/feature-finder.ts

```diff
@@ -122,7 +188,7 @@ export default class FeatureFinder {
     if (!location) {
       return;
     }
-    return {document, location};
+    return location;
```

Comment:
The `if` check above can be removed. I'd suggest removing the variable and just returning:
```js
return await this.getAstAtPosition(document, position);
```